libs/auth-api-lib/seeders/data/scope-iceland-health.ts

```diff
@@ -0,0 +1,12 @@
+import { createScope } from './helpers'
+
+export const up = createScope({
+  name: '@island.is/iceland-health',
+  displayName: 'Iceland Health applications',
+  description: 'Allow access to Iceland Health applications',
+
+  delegation: {
+    custom: true,
+  },
```

Comment:
Should also list procuringHolders if companies can also give custom delegations for this scope.